android-core/src/main/kotlin/com/mparticle/internal/BatchId.kt

```diff
@@ -0,0 +1,48 @@
+package com.mparticle.internal
+
+import com.mparticle.internal.database.services.MessageService.ReadyMessage
+
+class BatchId {
+    var mpid: Long
+        get() = field
```

Comment:
These `get()` methods should be able to be removed as well since they just return the value which the automatic getter will already do.